### DISCUSSION TITLE
I can't duplicate a profile

### DISCUSSION THREAD

**Discussion by aucejito** [microsoft/MixedRealityToolkit-Unity#10467] [Q&A]
I can't duplicate any Profile asset from MRTK because "The operation "Duplicate" cannot be executed because the selection or package is read-only.". Also, the buttons 'Copy and Customize' or 'Create a new profile' on the right are disabled.

![image](https://user-images.githubusercontent.com/38977418/153602296-6addee98-1f12-483b-8991-4e01d2d0bf3a.png)

I'm very new into this and I don't know how to solve it. Thanks in advance.

**Comment by hemasree99s** [comment]
Try clicking MixedRealityToolkit in the scene and then check if Customizing is enabled?

Also is the MRTK installed through Feature tool or through unity package?
